src/pages/components/map/mapComponent.jsx

```diff
@@ -29,13 +29,13 @@ export default class MapComponent extends Component {
             //style selected marker
 
             //first remove styles of previous marker
-            if (prevMarker) prevMarker.classList.remove('pin-active');
+            if (prevMarker) prevMarker.classList.remove('pin-active', 'pin-active-' + current.type);
```

Comment:
This throws an error if current is undefined: "TypeError: Cannot read property 'type' of undefined"

Reply:
![image](https://user-images.githubusercontent.com/8126481/79046276-465c5a80-7c10-11ea-83b3-ca3284f547ca.png)